internal/command/resume/worker.go

```diff
@@ -0,0 +1,42 @@
+package resume
+
+import (
+	"github.com/cirruslabs/orchard/pkg/client"
+	"github.com/spf13/cobra"
+)
+
+func newResumeVMCommand() *cobra.Command {
+	command := &cobra.Command{
+		Use:   "worker NAME",
+		Short: "Resume a worker",
+		RunE:  runResumeVM,
+		Args:  cobra.ExactArgs(1),
+	}
+
+	return command
+}
+
+func runResumeVM(cmd *cobra.Command, args []string) error {
```

Comment:
```suggestion
func runResumeWorker(cmd *cobra.Command, args []string) error {
```

Reply:
Renamed in https://github.com/cirruslabs/orchard/pull/73/commits/7fed7c2572967ec6143e86f9fd2fe712c5a1b41d.